### PR TITLE
Fix scripts, make `examples` `bin`'s

### PR DESCRIPTION
**Pull request by Buckram123**
This PR fixes outstanding  tasks on template

- [x] Move `examples` to `bin`
- [x] Update just commands to support multi-contracts
  - [x] Update publish-schema command
  - [x] Update publish command
- [x] Disable docs for adapter bins to avoid filename collision

### CODE REVIEW

**Comment by CyberHoward** [comment]
Why did we change the dir name (`app` -> `my-app`) ?

**Comment by Buckram123** [comment]
> Why did we change the dir name (`app` -> `my-app`) ?

Just so package name matches to the directory name. Also this section doesn't include renaming of the `app`/ https://github.com/AbstractSDK/app-template/blob/main/README.md#updating-names

File: justfile [hunk 1]
```diff
@@ -77,23 +77,23 @@ publish-schemas namespace name version: schema
...
   # check that the metadata exists
-  if [ ! -e "./metadata.json" ]; then \
+  if [ ! -e "./contracts/{{name}}/metadata.json" ]; then \
     echo "Please create metadata.json for module metadata"; exit; \
   fi
 
   tmp_dir="$(mktemp -d)"
-  schema_out_dir="$tmp_dir/{{namespace}}/{{name}}/{{version}}"
-  metadata_out_dir="$tmp_dir/{{namespace}}/{{name}}"
+  schema_out_dir="$tmp_dir/{{namespace}}/{{name}}/"
```
Comment (CyberHoward) [inline justfile:85]: This schema out dir includes the namespace while the schema.sh script doesn't seem to do that. Is that the expected behaviour?

**Comment by CyberHoward** [comment]
Oh and can we get rid of `const_format` as a deps? It's huge...

File: justfile (hunk 1 above)
Comment (Buckram123) [reply justfile:85]: Why would we need to include namespace in "app_template/schemas"? But we totally should include them when we push to the "AbstractSdk/schemas"

**Comment by Buckram123** [comment]
> Oh and can we get rid of `const_format` as a deps? It's huge...

Yeah we can remove it, we only use it in one place! What do you mean it's huge? Just tested compiled wasms with:
```
-rw-r--r--  448331 May 23 17:35 my_adapter.wasm
-rw-r--r--  424342 May 23 17:35 my_app.wasm
```
and without:
```
-rw-r--r--  448331 May 23 17:39 my_adapter.wasm
-rw-r--r--  424342 May 23 17:39 my_app.wasm
```

Could be nice to have abstract-based macro that generates those 3 consts - module-name, module-namespace and module-id and verify it in compile time.

**Comment by CyberHoward** [comment]
> > Oh and can we get rid of `const_format` as a deps? It's huge...
> 
> Yeah we can remove it, we only use it in one place! What do you mean it's huge? Just tested compiled wasms with:
> 
> ```
> -rw-r--r--  448331 May 23 17:35 my_adapter.wasm
> -rw-r--r--  424342 May 23 17:35 my_app.wasm
> ```
> 
> and without:
> 
> ```
> -rw-r--r--  448331 May 23 17:39 my_adapter.wasm
> -rw-r--r--  424342 May 23 17:39 my_app.wasm
> ```
> 
> Could be nice to have abstract-based macro that generates those 3 consts - module-name, module-namespace and module-id and verify it in compile time.

Yes, would love a custom macro for this that expands in all three variables. 

`abstract_app::module_id!("my_namespace", "my-module")`

Looked at the crate size and was very big. Guess compiler does a great job here at optimizing.

**Comment by Buckram123** [comment]
> > > Oh and can we get rid of `const_format` as a deps? It's huge...
> > 
> > 
> > Yeah we can remove it, we only use it in one place! What do you mean it's huge? Just tested compiled wasms with:
> > ```
> > -rw-r--r--  448331 May 23 17:35 my_adapter.wasm
> > -rw-r--r--  424342 May 23 17:35 my_app.wasm
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > and without:
> > ```
> > -rw-r--r--  448331 May 23 17:39 my_adapter.wasm
> > -rw-r--r--  424342 May 23 17:39 my_app.wasm
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Could be nice to have abstract-based macro that generates those 3 consts - module-name, module-namespace and module-id and verify it in compile time.
> 
> Yes, would love a custom macro for this that expands in all three variables.
> 
> `abstract_app::module_id!("my_namespace", "my-module")`
> 
> Looked at the crate size and was very big. Guess compiler does a great job here at optimizing.

Don't think there's much to optimize! We only use one macro from the crate, so only it's result ends up in the binary(formatted `str`)

**Pull request merged**